aws/eks-customer/versions.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_version = ">= 1.6"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.41.0"
+    }
+    # kubectl = {
+    #   source  = "gavinbunney/kubectl"
+    #   version = ">= 1.14.0"
+    # }
```

Comment:
please explain this

Reply:
This isn't needed. I forgot to remove it. thanks